src/Model/Item.php

```diff
@@ -243,6 +244,10 @@ public static function update(array $fields, array $condition)
 				$content_fields['raw-body'] = BBCode::removeAttachment($content_fields['raw-body']);
 
 				Post\Content::update($item['uri-id'], $content_fields);
+
+				$searchtext = Post\Engagement::getSearchTextForUriId($item['uri-id'], true);
+				DBA::update('post-engagement', ['searchtext' => $searchtext], ['uri-id' => $item['uri-id']]);
```

Comment:
Why do we have now two table with a FULLTEXT `searchtext` field and the `uri-id` key?

Reply:
Is this explained anywhere? What the “engagement” keyword mean to you?

Reply:
I explained the settings on the admin site setup page.